src/object/objecttype.c

```diff
@@ -573,10 +573,17 @@ cap_t createObject(object_t t, void *regionBase, word_t userSize, bool_t deviceM
 
 #ifdef CONFIG_KERNEL_MCS
     case seL4_SchedContextObject:
+        /** AUXUPD: "(True, ptr_retyps (refills_len (unat \<acute>userSize) (size_of TYPE(sched_context_C)) (size_of TYPE(refill_C)))
+          (Ptr ((ptr_val \<acute>regionBase) + word_of_nat (size_of TYPE(sched_context_C))) :: refill_C ptr)
+          \<circ> ptr_retyp (Ptr (ptr_val \<acute>regionBase) :: sched_context_C ptr))" */
```

Comment:
I realise the layout is copied from/consistent with the other ones, but line break is at 100 so we have a bit of space and it would be nicer to indent properly, i.e. the second line as a parameter to `ptr_retyps`, and the third line on the same semantic level as `ptr_retyps`.